code/API_definitions/network_access_management.yaml

```diff
@@ -0,0 +1,2863 @@
+          $ref: "#/components/responses/DefaultDeviceNotSupported403"
+        "404":
+          $ref: "#/components/responses/NetworkNotFound404"
+        "500":
+          $ref: "#/components/responses/Generic500"
+        "501":
+          $ref: "#/components/responses/Generic501"
+        "503":
+          $ref: "#/components/responses/Generic503"
+
+  /isolated-networks/{netId}/devices:
+    parameters:
+      - $ref: "#/components/parameters/netId"
+    get:
+      tags:
+        - Devices
+        - Isolated Networks
+      summary: Get all devices configured with a network
+      operationId: getDevicesByNetwork
+      description: |
+        Returns all devices configured with a network that matches the given ID.  Scopes impact the output of this endpoint.  See the examples for various permutations.
+
+        **Required Scopes:**
+          - **any of**
+            - `cpe-management-devices-read`
+            - `cpe-management-devices-write`
+            - `cpe-management-devices-hardware-address-read`
+          - **any of**
+            - `cpe-management-isolated-networks-read`
+            - `cpe-management-isolated-networks-write`
+            - `cpe-management-isolated-networks-primary-read`
+            - `cpe-management-isolated-networks-primary-write`
+            - `cpe-management-isolated-networks-auxiliary-read`
+            - `cpe-management-isolated-networks-auxiliary-write`
+
+        **Optional Scopes:**
+           - **any of**
+              - `cpe-management-service-sites-read`
+              - `cpe-management-service-sites-location-read`
+      responses:
+        "200":
+          description: Contains information about devices configured with the network
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DeviceList"
+              examples:
+                DeviceListWithNetworks:
+                  $ref: "#/components/examples/DeviceListWithNetworks"
+                DeviceListWithHardwareAddressAndNetworks:
+                  $ref: "#/components/examples/DeviceListWithHardwareAddressAndNetworks"
+                DeviceListWithServiceSiteAndNetworks:
+                  $ref: "#/components/examples/DeviceListWithServiceSiteAndNetworks"
+                DeviceListWithHardwareAddressAndServiceSiteAndNetworks:
+                  $ref: "#/components/examples/DeviceListWithHardwareAddressAndServiceSiteAndNetworks"
+        "400":
+          $ref: "#/components/responses/Generic400"
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/Generic403"
+        "404":
+          $ref: "#/components/responses/NetworkNotFound404"
+        "500":
+          $ref: "#/components/responses/Generic500"
+        "501":
+          $ref: "#/components/responses/Generic501"
+        "503":
+          $ref: "#/components/responses/Generic503"
+
+    patch:
+      tags:
+        - Devices

```

Comment:
```suggestion
```

Reply:
Keep devices since it is the resource modified.